styles.css

```diff
@@ -496,3 +501,104 @@ h2 {
   align-items: center;
   justify-content: space-between;
 }
+
+/* contact form  */
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/contact\ form\ shapes-mobile@2x.png"), url("images/Contact-me.png");
+  background-repeat: no-repeat;
+  background-position: right, center;
+  margin-top: 100px;
```

Comment:
- [ ] Please decrease the margin as it is shorter in the figma design:
![image](https://user-images.githubusercontent.com/91486791/220367021-6ecc1686-b221-4397-8f8b-c1f7b4849699.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/220367431-3895fe13-a418-404e-9524-bdfb9051a7c1.png)